test/cli/test/init.test.ts

```diff
@@ -0,0 +1,90 @@
+import { readdir, readFile, rm, writeFile } from 'node:fs/promises'
+import { join } from 'node:path'
+import { beforeEach, expect, test } from 'vitest'
+import { runVitestCli } from '../../test-utils'
+
+const ARROW_DOWN = '\u001B[B'
+const ENTER = '\n'
+
+const cwd = 'fixtures/browser-init'
+
+beforeEach(async () => {
+  await cleanup()
+  return cleanup
```

Comment:
why can't you just have the `cleanup` content in `beforeEach`?

Reply:
I'm not sure I understand - could you provide example?

Reply:
Ah, I see now you are calling the cleanup twice 🤔 

Why do you need to call it twice though?

Reply:
Well it should be enough to call it just as `afterEach`. But while I was developing this test case I had to hit `CTRL+c` couple of times and it always left the files on fs in that immediate termination.